backend/backend-python/debug_framework/cli/tensor_detector.py

```
@@ -0,0 +1,482 @@
+
+            # Search using patterns
+            for pattern in self.tensor_patterns:
+                try:
+                    for file_path in path_obj.rglob(pattern):
+                        if file_path.is_file():
+                            recording_info = self._analyze_tensor_file(file_path)
+                            if recording_info:
+                                detected_recordings.append(recording_info)
+                                if self.verbose:
+                                    print(f"✅ Found: {file_path.name} ({recording_info['size_mb']:.2f} MB)")
+                except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)